python/pyiceberg/io/pyarrow.py

```diff
@@ -278,6 +279,15 @@ def _get_fs(self, scheme: str) -> FileSystem:
                 "region": self.properties.get(S3_REGION),
             }
             return S3FileSystem(**client_kwargs)
+        elif scheme in {"gs", "gcs"}:
+            if access_token := self.properties.get("gs.access-token"):
+                client_kwargs = {
+                    "access_token": access_token,
+                    "credential_token_expiration": self.properties.get("gs.credential-token-expiration"),
+                }
+                return GcsFileSystem(**client_kwargs)
+            else:
+                return GcsFileSystem()
```

Comment:
[Question] Is it possible to take out this `if...else...`, like:
```python
        elif scheme in {"gs", "gcs"}:
            client_kwargs = {
                "access_token": self.properties.get("gs.access-token"),
                "credential_token_expiration": self.properties.get("gs.credential-token-expiration"),
            }
            return GcsFileSystem(**client_kwargs)
```
?
Is there any problem if `access_token` is not given but `credential_token_expiration` is provided? Thank you in advance for your answer.

Reply:
If `access_token` is set `credential_token_expiration` also needs to be set.